build/stackmat.js

```diff
@@ -317,7 +317,11 @@
       this.start = bind(this.start, this);
       this.signalFetched = bind(this.signalFetched, this);
       if (!supported()) {
-        alert("You need a recent browser in order to connect your Stackmat Timer.");
+        if (options != null ? options.onNonSuppotedBrowser : void 0) {
```

Comment:
Should be `options.onNonSupportedBrowser`, it's probably just missing a `npm run build`.